setup.py

```diff
@@ -185,6 +185,7 @@ def run(self):
             # The cloud acceptance test runner needs these
             "fabric==1.10.0",
             "apache-libcloud==0.16.0",
+            "digitalocean-python==0.1.5",
```

Comment:
New dependencies should go in the internal document with their licences. It might be helpful for future contributors if this is mentioned here in `setup.py`.